src/js/__test__/counter.test.js

```diff
@@ -0,0 +1,41 @@
+import loadBreeds from '../modules/load-counter-home';
+
+jest.mock('../modules/load-counter-home');
+
+describe('test counter function', () => {
+  describe('counter', () => {
+    test('must count 21 elements in homepage', () => {
+      document.body.innerHTML = '<ul class="list-container"></ul>';
+      // Arrange
+      const dogBreeds = [];
+      for (let i = 0; i < 200; i += 1) {
+        dogBreeds.push({
+          id: `${i}`,
+        });
+      }
+      const listContainer = document.querySelector('.list-container');
+      // Act
+      dogBreeds.forEach((breed) => {
+        breed.likeStatus = 'notLiked';
+        // Counter
+        if (document.querySelectorAll('li').length < 21) {
+          listContainer.innerHTML += `<li id="${breed.id}" class="breed-card">
+          <img class="breed-img" src="">
+          <div class="name-like-container">
+          <h2 class="names"></h2>
+          <div id="likes-container${breed.id}" class="likes-container">
+          <img id="like-icon${breed.id}" class="like-icon" src="">
+          <h4 id="likes-like-icon${breed.id}">0 Likes<h4/>
+          </div>
+          </div>
+          <button id="comment-${breed.id}" class="comment-button">Comments</button>
+          </li>`;
+        }
+      });
```

Comment:
This functionality is very similar to that of `load-counter-home.js#loadBreeds`. You should prefer testing that function since that's the one that's actually gonna be used on the production code.

Reply:
You could also add a test for a request-response under the hardcoded limit so that you test that the counter works well in that case.

Reply:
Ok :+1: